joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -1187,7 +1187,7 @@ class RegexDefinedFlowsDataFlowTests
     "be found" in {
       val src = cpg.identifier("Foo").l
       val snk = cpg.call("print").l
```

Comment:
This should produce a more representative flow
```suggestion
      val snk = cpg.call("print").argument(1).l
```